### ISSUE TITLE
[google_maps_flutter][android] - unable to focus on Marker when Talkback is ON

### ISSUE THREAD

**Issue by hellomentorplus** [flutter/flutter#116411]
## Use case
I turn on Talkback in my device, and try to navigate around the app using swipe to focus, but the marker is never focused, so I can't have any action with my marker

## Proposal
enable focus on all markers in SR mode

Video attached

https://user-images.githubusercontent.com/110004234/205280962-411fc3fc-4230-4c69-9fed-a80bebe134b6.mp4

**Comment by darshankawar** [comment]
@hellomentorplus 
Can you provide `flutter doctor -v` and minimal code sample that shows this behavior ?
